pkg/volume/gcepd/attacher.go

```diff
@@ -384,9 +384,29 @@ func (detacher *gcePersistentDiskDetacher) Detach(volumeName string, nodeName ty
 }
 
 func (detacher *gcePersistentDiskDetacher) UnmountDevice(deviceMountPath string) error {
+	if runtime.GOOS == "windows" {
+		// Flush data cache for windows because it does not do so automatically during unmount device
+		exec := detacher.host.GetExec(gcePersistentDiskPluginName)
+		err := writeVolumeCache(deviceMountPath, exec)
+		if err != nil {
+			klog.Errorf("writeVolumeCache (windows) failed with error %s", err)
```

Comment:
return the new error message instead of logging.  Let the caller decide to log the error

Reply:
fixed